unitary/alpha/quokka_sampler_test.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2024 The Unitary Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Iterable
+import pytest
+import cirq
+import sympy
+
+import unitary.alpha.quokka_sampler as quokka_sampler
+
+# Qubits for testing
+_Q = cirq.LineQubit.range(10)
+
+
+class FakeQuokkaEndpoint:
+    def __init__(self, responses: Iterable[quokka_sampler.JSON_TYPE]):
```

Comment:
for slightly nicer interface you can do `*responses: quokka_sampler.JSON_TYPE` and dispense with some square brackets below when passing expected responses. This would also remove the need for list conversion below

Reply:
Done.  Kept the list conversion so that I don't need to keep track of a counter or iterator.